lib/ansible/modules/apt.py

```diff
@@ -170,6 +170,11 @@
    - python3-apt (python 3)
    - aptitude (before 2.4)
 author: "Matthew Williams (@mgwilliams)"
+extends_documentation_fragment:
+- action_common_attributes
+attributes:
+    check_mode:
+        support: full
```

Comment:
This module also supports diff mode I think.

Reply:
good catch